data/coalition/coalition jobs.txt

```diff
@@ -1322,3 +1322,432 @@ mission "Coalition: From Games 3"
+
+
+mission "Hely Prisoner Transport"
+	job
+	repeat
+	name `Transport Resistance Prisoners`
+	description `Bring <bunks> Resistance members, who were arrested on <origin>, to <destination>.`
+	passengers 3 11
+	to offer
+		random < 10
+		has "license: Heliarch"
+	source
+		government "Coalition"
+	destination
+		government "Heliarch"
+	on offer
+		require "Enforcer Riot Staff"
+		require "Brig"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 20000 200
+		dialog `When you land on <destination>, Heliarch agents help you move the prisoners out of your ship. Once the last prisoner leaves, you're handed <payment>.`
+
+
+mission "Hely Investigate"
+	job
+	repeat
+	name `Investigate signs of Resistance activity`
+	description `Land on <stopovers>, where you and <bunks> other Heliarch agents will investigate an area where suspicious activity was reportedly seen. Payment is <payment>`
+	passengers 2 5
+	to offer
+		random < 20
+		has "license: Heliarch"
+	source
+		government "Heliarch"
+	stopover
+		distance 5 10
+	on visit
+		dialog phrase"generic passenger on visit"
+	on stopover
+		dialog `You and the other agents head to the reported area, looking for anything out of place. You don't find anything prominent, but there are clear signs that the place was recently used. You spend a few hours collecting evidence before heading back to your ship.`
+	on complete
+		payment
+		payment 90000
+		dialog `The agents thank you for transporting them in your ship before wishing you safe travels, heading off to deliver your findings to their superiors. You're transferred <payment> a few moments later.`
+
+
+mission "Evacuate Resistance True"
+	name "Rescue Resistance Members"
+	description "Pick up the <bunks> Resistance members on <stopovers>, then bring them to <destination>."
+	deadline
+	repeat
+	minor
+	to offer
+		random < 3
+		has "Heliarch Interrogation 2: declined"
+		has "pledged to resistance"
+		not "Evacuate Resistance True: active"
+		not "Evacuate Resistance Fake: active"
+		not "assisted heliarch"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch Containment 4-A: offered"
+	to fail
+		has "Heliarch Investigation 2: active"
+	passengers 24 65
+	source
+		government "Coalition"
+	stopover
+		government "Coalition"
+		distance 2 6
+	destination
+		government "Coalition"
+		distance 3 5
+	on offer
+		dialog `You wander around the spaceport for a bit, without finding anything of interest. Once you get back to your ship, you receive a message from the Resistance: <bunks> members need someone to rescue them on <stopovers>, where the Heliarchs are investigating, and transport them to <destination>.`
+	on stopover
+		dialog `The Resistance members rush inside your ship, urging you to take off quickly.`
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		dialog `The Resistance members thank you for rescuing them before handing you <payment>.`
+		payment 23861 8
```

Comment:
I don't know a lot about syntax here, but the space seems strange. Wanted to point it out and confirm it was intentional.